samples/kotlin-android-app-destinations/src/main/java/com/segment/analytics/destinations/EventFragment.kt

```diff
@@ -109,10 +109,12 @@ class EventFragment(val type: EventType, val analytics: Analytics) : Fragment()
         container.addView(view)
     }
 
-    private inline fun <reified T : BaseEvent> eventStr(event: T) = Json {
+    private val Json = Json {
```

Comment:
nit: what's the consideration behind declaring a separate variable? it seems to be only used once and it's private. I think it's probably better to make it a functional level variable:
```kotlin
    private inline fun <reified T : BaseEvent> eventStr(event: T): String{
        val json = Json {
            prettyPrint = true
            encodeDefaults = true
        }
        
        return json.encodeToString(event)
    }
```

Reply:
this would put us in the same place as before. What we are trying to do is to avoid constructing `Json { ... }` each time, and having it inside the function will in turn mean that everytime `evenStr` is called, we create a new copy of that object.

Reply:
ah. totally make sense 👍